### ISSUE TITLE
Improve util.Counter

### ISSUE THREAD

**Issue by bonn2** [bonn2/ElementalArmor#13]
It would be great if the size of the bar that counter draws was made dynamic. This would allow players to easily distinguish between each counter's value. And open up the opportunity for multilevel charms, like Elytra Boost 2 which gives you 6seconds as opposed to 3

**Comment by Jaimss** [comment]
I think maybe add a constructor param that will be the name of the counter, so you can display stuff like
```
Elytra Boost » [ |||||||||| ]
Elytra Boost 2 » [ |||||||||||||||||||| ]
```
etc etc

**Comment by bonn2** [comment]
Yeah, that would also be great. I'll get working on it now

**Comment by bonn2** [comment]
One final idea, what if the number of bars was automatically calcualted based on the max size (unless predefined) like max size 40 = 10 bars?

**Comment by bonn2** [comment]
Implemented in commit 60d0a8bc305e316121cbe244b6682e5689cc6bcd
